### ISSUE TITLE
Thanks for your great contribution!

### ISSUE THREAD

**Issue by zqgao22** [evolutionaryscale/esm#53]
Hi, I would like to ask, how should I generate (create) `sequence_id` if I want to fine-tune or retrain ESM-3? Could you kindly provide a specific example, such as if the batch size is 2 and sequence lengths are 3 and 6? In this case, what should this 2x6 tensor of `sequence_id` look like?

Thanks very very much.

Best regards.

**Comment by ebetica** [comment]
This is a left-over artifact from our internal models for bin-packing to improve efficiency of BERT training. It might work but I haven't tested it fully. I suggest just concating your sequences along the batch dimension with the proper padding. In this case, you can just set the `sequence_id` to the padding mask.
